Tests/PackageLoading/InMemoryConventionTests.swift

```diff
@@ -0,0 +1,174 @@
+import XCTest
```

Comment:
I'd probably just call this convention tests, the in memory part doesn't need to be part of the name (it will just become the default, and we should actively migrate to it).